src/admin-views/admin-welcome-message.php

```diff
@@ -2,84 +2,213 @@
+	<img
+		class="tribe-events-admin-graphic tribe-events-admin-graphic--mobile-only"
+		<?php $mobile_graphic = $has_plus ? 'images/header/welcome-mobile-etplus.jpg' : 'images/header/welcome-mobile-et.jpg'; ?>
+		src="<?php echo esc_url( tribe_resource_url( $mobile_graphic, false, null, $main ) ); ?>"
+		alt="<?php esc_attr_e( 'Shapes and lines for visual interest', 'event-tickets' ); ?>"
+	/>
 
-	<div class="tribe-half-column">
-		<h4 data-tribe-icon="dashicons-welcome-learn-more"><?php esc_html_e( 'Getting Started', 'event-tickets' ); ?></h4>
-		<p><?php esc_html_e( 'Start strong with these helpful resources.', 'event-tickets' ); ?></p>
-		<ul>
-			<li><a href="https://theeventscalendar.com/product/wordpress-event-tickets/" target="_blank"><?php esc_html_e( 'Key Features', 'event-tickets' ); ?></a></li>
-			<li><a href="http://m.tri.be/1a9u" target="_blank"><?php esc_html_e( 'Settings Overview', 'event-tickets' ); ?></a></li>
-			<li><a href="http://m.tri.be/1a9v" target="_blank"><?php esc_html_e( 'New User Primer', 'event-tickets' ); ?></a></li>
-		</ul>
+	<div class="tribe-events-admin-title">
+		<img
+			class="tribe-events-admin-title__logo"
+			<?php $logo_image = $has_plus ? 'images/logo/event-tickets-plus.svg' : 'images/logo/event-tickets.svg'; ?>
+			src="<?php echo esc_url( tribe_resource_url( $logo_image, false, null, $main ) ); ?>"
+			alt="<?php esc_attr_e( 'Event Tickets logo', 'event-tickets' ); ?>"
+		/>
+		<h2 class="tribe-events-admin-title__heading"><?php
+			if ( $has_plus ) :
+				esc_html_e( 'Event Tickets Plus', 'event-tickets' );
+			else :
+				esc_html_e( 'Event Tickets', 'event-tickets' );
+			endif;
+		?></h2>
+		<p class="tribe-events-admin-title__description"><?php
+			if ( $has_plus ) :
+				esc_html_e( 'Thanks for installing Event Tickets Plus! Here are some handy resources for getting started with our plugins.', 'event-tickets' );
+			else :
+				esc_html_e( 'Thanks for installing Event Tickets! Here are some handy resources for getting started with our plugins.', 'event-tickets' );
+			endif;
+		?></p>
 	</div>
 
-	<div class="tribe-half-column">
-		<h4 data-tribe-icon="dashicons-sos"><?php esc_html_e( 'Resources and Support', 'event-tickets' ); ?></h4>
-		<p><?php esc_html_e( 'We’ve got your back every step of the way.', 'event-tickets' ); ?></p>
-		<ul>
-			<li><a href="http://m.tri.be/1a9w" target="_blank"><?php esc_html_e( 'Search the Knowledgebase', 'event-tickets' ); ?></a></li>
-			<li><a href="http://m.tri.be/1a9x" target="_blank"><?php esc_html_e( 'Available Translations', 'event-tickets' ); ?></a></li>
-			<li><a href="http://m.tri.be/1a9y" target="_blank"><?php esc_html_e( 'Submit a Help Desk Request', 'event-tickets' ); ?></a></li>
+	<div class="tribe-events-admin-quick-nav">
+		<div class="tribe-events-admin-quick-nav__title"><?php esc_html_e( 'Quick Links:', 'event-tickets' ); ?></div>
+		<ul class="tribe-events-admin-quick-nav__links">
+			<?php if ( $has_plus && $has_tec ) : // ET+ with TEC ?>
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 0ccef62df179c38d9acec34bf9b5b2b6</sub>